### DISCUSSION TITLE
missing authority table - non-publication-identifiers

### DISCUSSION THREAD

**Discussion by odscjen** [OP-TED/eForms-SDK#155] [Q&A] [answered]
The Unpublished Identifier, BT-195 (efbc:FieldIdentifierCode) references `listName="non-publication-identifier"`. In the rest of the guidance when a listName is given like this it seems to reference one of the authority tables listed [here](https://op.europa.eu/en/web/eu-vocabularies/authority-tables). But I can't see one for the non publication identifiers? Will this table be getting created at a later date?

**Comment by jpmckinney** [comment, marked as answer]
Some codelists are local to the SDK, including this one: https://github.com/OP-TED/eForms-SDK/blob/develop/codelists/non-publication-identifier.gc
